### PR TITLE
Fix #1533: The Predictoors Table

### LINKED ISSUE #1533: [PdrDashboard, v0.5] The Predictoors Table
Create a table that contains all the columns from the mockup provided bellow

image

### PR DESCRIPTION
**Pull request by kdetry**
Fixes #1533

### CODE REVIEW

File: pdr_backend/pdr_dashboard/pages/predictoors.py [hunk 1]
```diff
@@ -100,29 +101,24 @@ def get_search_bar_row(self):
 
     def get_main_container(self):
         # TODO: replace with predictoors data
```
Comment (calina-c) [inline pdr_backend/pdr_dashboard/pages/predictoors.py:103]: I think you can remove this TODO now

File: pdr_backend/pdr_dashboard/pages/predictoors.py [hunk 2]
```diff
@@ -132,108 +128,46 @@ def get_feeds_table_area(
...
-                feed["contract"], feed_subcription_stats
-            )
+        ## split the pair column into two columns
+        for data_item in temp_data:
+            print("data_item--->", data_item)
 
-            feed_item.update(subscription_result)
+            temp_pred_item = {}
+            temp_pred_item["addr"] = str(data_item["user"])
+            temp_pred_item["gross_income_(OCEAN)"] = data_item["total_payout"]
```
Comment (calina-c) [inline pdr_backend/pdr_dashboard/pages/predictoors.py:156]: should we remove the (OCEAN) part?

**Review by calina-c — CHANGES_REQUESTED** [review]
just a few minor comments, and please apply black formatting

File: pdr_backend/pdr_dashboard/pages/predictoors.py (hunk 2 above)
Comment (kdetry) [reply pdr_backend/pdr_dashboard/pages/predictoors.py:156]: According to the task, there should be the `(OCEAN)` phrase on the column header

https://github.com/oceanprotocol/pdr-backend/issues/1533

Comment (calina-c) [reply pdr_backend/pdr_dashboard/pages/predictoors.py:153]: on the column header, yes... but is it really necessary as a data key? With filtering and formatting and all that, I think adding parantheses (which are special characters) is maybe not such a good idea

Comment (calina-c) [reply pdr_backend/pdr_dashboard/pages/predictoors.py:153]: I'm fine either way, just asking for your opinion too

Comment (kdetry) [reply pdr_backend/pdr_dashboard/pages/predictoors.py:153]: I can agree that it is not pretty 👍 we used like that on the feeds table too

**Pull request merged**